gnomad_qc/v4/assessment/summary_stats.py

```diff
@@ -0,0 +1,230 @@
+                    ht.region_flags.outside_ukb_capture_region
+                    | ht.region_flags.outside_broad_capture_region
+                ),
+            )
+
+            mt = default_generate_gene_lof_matrix(
+                mt=mt,
+                tx_ht=None,
+                additional_grps=[
+                    "broad_ukb_union_intervals",
+                    "broad_ukb_intersection_intervals",
+                    "fail_interval_qc",
+                ],
+            )
+            mt.write(
+                release_lof(test=test, data_type="exomes", mt=True).path,
+                overwrite,
+            )
+
+        if args.summarize_gene_lof_matrix:
+            mt = release_lof(test=test, data_type="exomes", mt=True).mt()
+            mt = mt.annotate_cols(
+                meta=mt.meta.annotate(pop=mt.meta.population_inference.pop)
+            )
+            ht = default_generate_gene_lof_summary(mt)
+            ht.write(
+                release_lof(test=test, data_type="exomes").path,
+                overwrite,
+            )
+
+    finally:
+        logger.info("Copying log to logging bucket...")
+        hl.copy_log(get_logging_path("summary_stats"))
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "-o",
+        "--overwrite",
+        help="Overwrite data (default: False)",
+        action="store_true",
+    )
+    parser.add_argument(
+        "--slack-channel", help="Slack channel to post results and notifications to."
+    )
+    parser.add_argument(
+        "--test",
+        help="Test on a small number of variants",
+        action="store_true",
+    )
+    parser.add_argument(
+        "--get-summary-counts",
+        help="Get summary counts per variant category.",
+        action="store_true",
+    )
+    parser.add_argument(
+        "--interval-qc-pass-only",
+        help="Generate summary stats on interval QC pass regions only.",
+        action="store_true",
+    )
+    parser.add_argument(
+        "--union-calling-interval-only",
+        help=(
+            "Generate summary stats on regions in the union between UKB and Broad "
+            "intervals only."
+        ),
+        action="store_true",
+    )
+    parser.add_argument(
+        "--intersection-calling-interval-only",
+        help=(
+            "Generate summary stats on regions in the intersection between UKB and "
+            "Broad intervals only."
+        ),
+        action="store_true",
+    )
+    parser.add_argument(
+        "--use-100k-downsampling",
+        help="Use the 100K downsampling for summary stats.",
+        action="store_true",
+    )
+
+    parser.add_argument(
+        "--generate-gene-lof-matrix",
+        help="Generate gene LoF matrix.",
+        action="store_true",
+    )
+    parser.add_argument(
+        "--summarize-gene-lof-matrix",
+        help="Creates gene LoF matrix summary Table.",
+        action="store_true",
+    )
```

Comment:
parser.add_argument(
        "--data-type",
        default="exomes",
        choices=["exomes","genomes"],
        help="Data type (exomes or genomes) to produce summary stats for."
    )